examples/body-scroll.html

```diff
@@ -0,0 +1,94 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+
+  <title>Hyper List Demo</title>
+
+  <style>
+    body, html {
+      padding: 0;
+      margin: 0;
+      height: 100%;
+      width: 100%;
+      min-height: 100%;
+      min-width: 100%;
+      text-align: center;
+    }
+    body {
+      overflow: hidden;
+    }
+
+    @media
+    only screen and (-webkit-min-device-pixel-ratio : 1.5),
+    only screen and (min-device-pixel-ratio : 1.5) {
+      .container {
+        width: 100%;
+        height: 100%;
+        min-height: 100%;
+      }
+    }
+
+    .vrow {
+      width: 100%;
+      height: 30px;
+      max-height: 30px;
+      color: #000;
+      margin: 0;
+      padding-top: 10px;
+      padding-bottom: 10px;
+    }
+
+    .vrow p {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      margin: 0;
+      color: #5b5b5b;
+    }
+  </style>
+</head>
+<body>
+
+<header>
+  <h1>Hyperlist</h1>
+</header>
+<div id="list">
+</div>
+<footer>
+  github
+</footer>
+
+<script type="text/javascript" src="../dist/hyperlist.js"></script>
+<script>
+  var container = document.getElementById('list');
+  var config = {
+    height: window.innerHeight,
+    itemHeight: 50,
+    total: 100000,
+    // Set to true to put into 'chat mode'.
+    reverse: false,
+    scrollerTagName: 'div',
+    scrollContainer: document.body,
+    overrideScrollPosition() {
+      return window.pageYOffset
```

Comment:
Should be `return (window.pageYOffset - (container.offsetTop - scrollContainer.offsetTop)) || 0` to return valid `scrollContainer` scroll position